### PR TITLE
Move remaining member rude edit analysis to semantics

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@davidwengier PTAL

File: src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs [hunk 1]
```diff
@@ -14139,22 +14188,6 @@ public void Parameter_Attribute_Insert()
                 capabilities: EditAndContinueTestHelpers.Net6RuntimeCapabilities);
         }
 
-        [Fact]
-        public void Parameter_Attribute_Insert_SupportedByRuntime_NonCustomAttribute()
```
Comment (davidwengier) [inline src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs:14143]: Why delete this one? I don't think there is another test that covers the deny list of attributes

File: src/Features/Core/Portable/EditAndContinue/EditAndContinueDiagnosticDescriptors.cs [hunk 2]
```diff
@@ -96,8 +96,6 @@ void AddGeneralDiagnostic(EditAndContinueErrorCode code, string resourceName, Di
             AddRudeEdit(RudeEditKind.InsertGenericMethod, nameof(FeaturesResources.Adding_a_generic_0_will_prevent_the_debug_session_from_continuing));
             AddRudeEdit(RudeEditKind.Move, nameof(FeaturesResources.Moving_0_will_prevent_the_debug_session_from_continuing));
             AddRudeEdit(RudeEditKind.Delete, nameof(FeaturesResources.Deleting_0_will_prevent_the_debug_session_from_continuing));
-            AddRudeEdit(RudeEditKind.MethodBodyAdd, nameof(FeaturesResources.Adding_a_method_body_will_prevent_the_debug_session_from_continuing));
```
Comment (davidwengier) [inline src/Features/Core/Portable/EditAndContinue/EditAndContinueDiagnosticDescriptors.cs:99]: Delete the resources?

**Review by davidwengier — APPROVED** [review]
:shipit:

File: src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs (hunk 1 above)
Comment (tmat) [reply src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs:14143]: Mistake. Reverted.

**Pull request merged**